lib/features/app/di/app_scope.dart

```diff
@@ -62,16 +104,46 @@ class AppScope implements IAppScope {
     _dio = _initDio(additionalInterceptors);
     _errorHandler = DefaultErrorHandler();
     _router = AppRouter.instance();
+    _secureStorage = const FlutterSecureStorage();
+    _tokensStorage = TokensStorage(_secureStorage);
+    _firstRunStorage = FirstRunStorage(_sharedPreferences);
     _themeModeStorage = ThemeModeStorageImpl(_sharedPreferences);
     _analyticsService = AnalyticsServiceImpl([
       // TODO(init): can be removed MockFirebaseAnalytics and MockAmplitudeAnalytics, added for demo analytics track
       FirebaseAnalyticTracker(MockFirebaseAnalytics()),
       AmplitudeAnalyticTracker(MockAmplitudeAnalytics()),
     ]);
+
+    // TODO(anyone): needs customization for a specific project
```

Comment:
Could we make a template implementation for this? I tried to figure it out, but honestly, I couldn't do it by following the instructions. I don't understand where and what needs to be added